pom.xml

```diff
@@ -439,16 +440,16 @@
 
 					<includes>
 						<include>${gpuTestsPath}</include> <!-- Path for GPU integration tests, enabled for gpuTests profile -->
-						<include>**/integration/applications/**/*Suite.java</include>
-						<include>**/integration/conversion/*Suite.java</include>
-						<include>**/integration/functions/data/*Suite.java</include>
+						<!-- <include>**/integration/applications/**/*Suite.java</include> -->
+						<!-- <include>**/integration/conversion/*Suite.java</include> -->
+						<!-- <include>**/integration/functions/data/*Suite.java</include> -->
 						<include>**/integration/functions/sparse/*Suite.java</include>
-						<include>**/integration/functions/codegenalg/*Suite.java</include>
-						<include>**/integration/functions/**/*Test*.java</include>
+						<!-- <include>**/integration/functions/codegenalg/*Suite.java</include> -->
+						<!-- <include>**/integration/functions/**/*Test*.java</include> -->
 						<include>**/integration/mlcontext/*Suite.java</include>
-						<include>**/integration/mlcontext/algorithms/*Suite.java</include>
+						<!-- <include>**/integration/mlcontext/algorithms/*Suite.java</include> -->
 						<include>**/integration/scripts/nn/*Suite.java</include>
-						<include>**/integration/scalability/**/*Test.java</include>
+						<!-- <include>**/integration/scalability/**/*Test.java</include> -->
```

Comment:
I am assuming everybody is ok with removing these tests. What are the implications around test coverage?

Reply:
"integration/scalability" has been removed a long time ago from our testsuite (and moved to applications) - this is a good opportunity to clean up the pom

Reply:
we have very low coverage now, but will add more once we switched. For now, we just want to stay within the travis-ci limits.